ts/features/walletV3/payment/screens/WalletPaymentPickMethodScreen.tsx

```diff
@@ -124,6 +177,7 @@ const WalletPaymentPickMethodScreen = () => {
       // currently we do not allow onboarding new methods in payment flow
       // dispatch(walletPaymentGetAllMethods.request());
       dispatch(walletPaymentGetUserWallets.request());
+      dispatch(walletPaymentResetPspList());
```

Comment:
Why reset the list if we already have the data?
Also this completely remove the sense to fetch PSPs from the method selection screen. What do you think?